### ISSUE TITLE
Bug Report: Repo shows wrong URL

### ISSUE THREAD

**Issue by bdorney** [cms-gem-daq-project/cmsgemos#267]
## Brief summary of issue

RPM shows wrong URL after build.

### Types of issue

- [x] Bug report (report an issue with the code)
- [ ] Feature request (request for change which adds functionality)

## Expected Behavior

Would expect `URL` to point to `cmsgemos` version.

## Current Behavior

```bash
[dorney@gem904qc8daq]~/scratch0/CMS_GEM/CMS_GEM_DAQ/cmsgemos% yum info \*cmsgemos\*gempython\*
Loaded plugins: changelog, fastestmirror, kernel-module, langpacks, protectbase, tsflags, versionlock
Loading mirror speeds from cached hostfile
252 packages excluded due to repository protections
Installed Packages
Name        : cmsgemos_gempython
Arch        : x86_64
Version     : 1.0.26
Release     : 1.0.45.dev.2fc66e6git.cc7.python2.7
Size        : 361 k
Repo        : installed
Summary     : None
URL         : https://cms-gem-daq-project.github.io/gem-plotting-tools
License     : MIT
Description : UNKNOWN
```

### Steps to Reproduce (for bugs)

1. Build the package

## Context

Not really affected...

## Your Environment

* Version used: 2417b5bcdd2f468a3c69637474f3f97eb1709565
* Shell used:

**Comment by lpetre-ulb** [comment]
Could it be fixed in _legacy_? The change seems to be trivial and the official website is now available, not with the final URL, though AFAIK.

**Comment by jsturdy** [comment]
easily, I'll add it to the docs changes in #339 (also in other packages), the options for what to put in there are:
* github repo link
* API docs link
* generic gemdaq landing

**Comment by lpetre-ulb** [comment]
Thanks! I would use the generic gemdaq landing page as URL in the RPM packages, so that it would to point the (power) users to the official website/documentation. The other possibilities (GitHub repo and API documentation) should easily be accessible from the main website.

**Comment by jsturdy** [comment]
Fixed in #339
